docs/en-US/about_Plaster_CreatingAManifest.help.md

```diff
@@ -267,15 +289,17 @@ This example shows Plaster parameter expansion working in the message element:
 ```
 
 ### Content element: Modify
-The modify element allows you to replace file contents, allowing you to copy files using the `file` element, then substituting content to meet your needs.
+The modify element allows you to replace file contents, allowing you to copy files using the `file` element, then substituting content to meet your needs. Multiple original, substitute and condition attributes can be used in a single modify element to modify a file to your requirements.
 
```

Comment:
I think that should read "Multiple replace elements can be used in a single modify element.  Each replace element has one original and one substitute element to define what that replace operation does to the file.